projects/grafana/helm-charts/Makefile

```diff
@@ -0,0 +1,30 @@
+GRAFANA_CHART_REPO=helm-charts
+GRAFANA_CHART_CLONE_URL=https://github.com/grafana/$(GRAFANA_CHART_REPO).git
+GRAFANA_CHART_GIT_TAG=grafana-6.4.4
```

Comment:
Can you add conditions to check these 3 values are not empty, like we do [here](https://github.com/aws/eks-distro-build-tooling/blob/main/projects/prometheus-community/helm-charts/Makefile#L10-L12)?

Reply:
sure, but why do we need to check it if we're setting it?